extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/authentication/db/updater/CoordinatorBasicAuthenticatorMetadataStorageUpdater.java

```diff
@@ -91,7 +91,7 @@ public CoordinatorBasicAuthenticatorMetadataStorageUpdater(
       BasicAuthCommonCacheConfig commonCacheConfig,
       @Smile ObjectMapper objectMapper,
       BasicAuthenticatorCacheNotifier cacheNotifier,
-      ConfigManager configManager // ConfigManager creates the db table we need, set a dependency here
+      ConfigManager configManager // -V6022: ConfigManager creates the db table we need, set a dependency here
```

Comment:
What does `-V6022` mean?

Reply:
It means an unused parameter. I've extended the comment.